### ISSUE TITLE
LR Scheduler Understanding

### ISSUE THREAD

**Issue by sumitmishra27598** [facebookresearch/fairseq#4338]
Let's say I trained the fairseq model with an initial learning rate of 1e-4 along with poly decay and trained the model for 2 epochs.
Saved checkpoints.

Then, after some time I want to continue training using saved checkpoint, so do I need to change the learning rate or it will be there in the saved checkpoint, and it will take the last used learning rate only?

I only need this understanding.
